src/main/java/hu/oe/nik/szfmv/automatedcar/systemcomponents/Ultrasonic.java

```diff
@@ -107,4 +107,41 @@ private void getAllCollidablesInRange() {
         getAllCollidablesInRange();
         return collidableObjectsInRange;
     }
+
+    public void searchParkingSpanceBasedonIndex(){
+        String indexStatus =  virtualFunctionBus.guiInputPacket.getIndexStatus().toString();
+
+        if (indexStatus == "LEFT")
+        {
+            //parkolohely kereésé
+            for (Sensor sensor : sensors) {
+                if (sensor.getSensorPosition() == UltrasoundPositions.REAR_LEFT_SIDE || sensor.getSensorPosition() == UltrasoundPositions.FRONT_LEFT_SIDE)
+                {
+                    for (WorldObject object : world.getObjectsInsideTriangle(sensor.getTriangleSource(),
+                            sensor.getTriangleCorner1(), sensor.getTriangleCorner2())) {
+
+                        if(object.getType() == "parking_space_parallel"){
+
+                            //hely hosszát néző metodus NFC osztaly kell
```

Comment:
Ez NPC akar lenni?

Reply:
@chowder23 Jobban belegondolva nem is értem a mondatot, az NFC előtt egy vessző van?
Ha igen, akkor igazából nem kell várni ám a NPC user story-ra (#7 ), mert az valójában csak a mozgó NPC-kkel foglalkozik. Statikusan autót ti is letehettek autót a parkolóba.

Reply:
@bodapeter írta ezt. Szerintem NPC lenne és vessző előtte.

Reply:
Oké, azt már nem néztem meg.